frontend/package.json

```diff
@@ -9,33 +9,33 @@
     "build:frontend": "webpack --config webpack/webpack.prod.ts"
   },
   "dependencies": {
-    "@emotion/react": "^11.9.0",
-    "@emotion/styled": "^11.8.1",
-    "@mui/icons-material": "^5.6.2",
-    "@mui/material": "^5.6.4",
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.9.1",
     "js-cookie": "^3.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-query": "^3.39.1"
+    "react-query": "^3.39.2"
   },
   "devDependencies": {
     "@types/js-cookie": "^3.0.2",
-    "@types/node": "^17.0.35",
-    "@types/react": "^18.0.9",
-    "@types/react-dom": "^18.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.26.0",
-    "@typescript-eslint/parser": "^5.26.0",
+    "@types/node": "^17.0.45",
+    "@types/react": "^17.0.47",
+    "@types/react-dom": "^17.0.17",
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^5.30.7",
     "css-loader": "^6.7.1",
-    "eslint": "^8.16.0",
-    "eslint-plugin-react": "^7.30.0",
-    "eslint-plugin-react-hooks": "^4.5.0",
-    "mini-css-extract-plugin": "^2.6.0",
-    "ts-loader": "^9.3.0",
-    "ts-node": "^10.8.0",
-    "typescript": "^4.6.4",
-    "webpack": "^5.72.1",
-    "webpack-bundle-tracker": "1.5.0",
-    "webpack-cli": "^4.9.2",
+    "eslint": "^8.20.0",
+    "eslint-plugin-react": "^7.30.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "mini-css-extract-plugin": "^2.6.1",
+    "ts-loader": "^9.3.1",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.7.4",
```

Comment:
I upgraded TS, which is slightly riskier update, but I checked the release notes to ensure none of the breaking changes would affect us: https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#breaking-changes

Reply:
Okay, sounds good!